test/test_stdout.rb

```diff
@@ -18,4 +18,9 @@ def test_default_list_output
       assert_match(/.*#{spec_key}.*/, @stdout.string)
     end
   end
+
+  def test_ubuntu_host_generation_output
```

Comment:
So in this test I'm asserting the contents of @stderr, not $stderr, and in the setup phase of the test @stderr is initialised as `@stderr = StringIO.new`. I just chose the Ubuntu string because I knew it would hit the warning that gets written to stderr, should I just replace it with a simpler token? Something like `centos6-64ma`?